### PR TITLE
bkt 0.5.2 (new formula)

### PR DESCRIPTION
**Pull request by zbentley**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] ~**Note: does not pass brew audit yet due to insufficient stars. It's growing in popularity so I expect this to be mergeable soon. **~ your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

### CODE REVIEW

**Comment by carlocab** [comment]
This needs to be in the `Formula` subdirectory and not the root of the repository.

**Comment by zbentley** [comment]
@carlocab requesting re-review on this when you have time. Thanks!

**Comment by carlocab** [comment]
Thanks -- can you rebase (i.e. `git rebase origin/master && git push --force`) your PR branch? PRs with merge commits can't be merged because of the linear history requirement. Squashing your commits together will also help since I suspect @BrewTestBot will get confused from moving the formula into the `Formula` subdirectory.

**Comment by zbentley** [comment]
@carlocab I ran that command but it seems to have pretty thoroughly trashed this branch. Is this what you're after or shall I open another branch/PR?

**Review by carlocab — DISMISSED** [review]
Fixed that up for you.

**Comment by zbentley** [comment]
> Fixed that up for you.

Thanks!

File: Formula/bkt.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+class Bkt < Formula
+  desc "Utility for caching the resilts of shell commands"
```
Comment (dawidd6) [inline Formula/bkt.rb:2]:
```suggestion
  desc "Utility for caching the results of shell commands"
```

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**